From 3ed7474364eebd01beefd76faa752cf4c6dc88a2 Mon Sep 17 00:00:00 2001
From: noaione <noaione0809@gmail.com>
Date: Thu, 10 Oct 2024 09:43:59 +0700
Subject: build: fix macOS build


diff --git a/unrarsrc/makefile b/unrarsrc/makefile
index ce54a02..0d3872e 100644
--- a/makefile
+++ b/makefile
@@ -3,7 +3,7 @@
 
 # Linux using GCC
 CXX=c++
-CXXFLAGS=-march=native -O2 -std=c++11 -Wno-logical-op-parentheses -Wno-switch -Wno-dangling-else
+CXXFLAGS=-std=c++11 -arch=native -O2 -std=c++11 -Wno-logical-op-parentheses -Wno-switch -Wno-dangling-else
 LIBFLAGS=-fPIC
 DEFINES=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DRAR_SMP
 STRIP=strip
